### PR TITLE
Loading state on submitting membership form

### PR DESCRIPTION
**Pull request by merissali**
🎟️ Ticket(s): Closes #

👷 Changes: When a user presses 'submit', the submit button is greyed out & disabled so the user can no longer click it, until the backend call resolves & it brings them to the 'signup success' page. (Before, it would just show nothing so the user didn't know if it was submitting. This lead them to just spam the button, which would then result in error messages of 'A user with this email already exists' even though they were signing up for the first time.)

💭 Notes: 

Wait! Before you merge, have you checked the following:

📷 Screenshots

Screen Shot 2021-12-05 at 10 00 28 AM

## Checklist

- [ ] Looks good on large screens
- [ ] Looks good on mobile

### CODE REVIEW

**Comment by huangsamantha** [comment]
Hihi! This looks good, but I wonder if it might be less disruptive to just change the submit button while it's submitting? Like fade it out as if it's been clicked.

**Comment by merissali** [comment]
Hi @huangsamantha I updated it so the button is grey after pressing submit (see screenshot), and since it's disabled the cursor changes from a hand cursor to a pointer cursor (and they can't click the button). Thoughts?

File: src/pages/public/MembershipForm/index.js [hunk 1]
```diff
@@ -132,9 +132,12 @@ const MembershipFormContainer = (props) => {
       admin,
     };
 
+    setIsSubmitting(true);
+    // alert("Submitting...");
```
Comment (pzhang24) [inline src/pages/public/MembershipForm/index.js:136]: Remove this comment?

File: src/pages/public/MembershipForm/index.js [hunk 2]
```diff
@@ -144,9 +147,13 @@ const MembershipFormContainer = (props) => {
         } else {
           console.log(err);
         }
+        setIsSubmitting(false);
       });
   }
 
+  // if (isSubmitting) {
```
Comment (pzhang24) [inline src/pages/public/MembershipForm/index.js:154]: These as well

**Review by pzhang24 — COMMENTED** [review]
Looking good! Could you clean up some of these comments in the code, and then I'll approve it

**Review by pzhang24 — APPROVED** [review]
Looks good!!

**Pull request merged**